src/libraries/adaptative-curve/ConstantsLib.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+library ConstantsLib {
+    /// @notice Maximum rate at target per second (scaled by WAD) (1B% APR).
+    int256 internal constant MAX_RATE_AT_TARGET = int256(0.01e9 ether) / 365 days;
+
+    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
+    int256 internal constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
```

Comment:
so do we use this value in the end cc @MathisGD @QGarchery

Reply:
The min rate at target has always been 0.1% and I suggested to use 0.01% but it was denied in #64